core/src/com/unciv/logic/map/MapParameters.kt

```diff
@@ -131,6 +131,8 @@ object MapType : IsPartOfGameInfoSerialization {
     const val default = "Default"
     const val pangaea = "Pangaea"
     const val continents = "Continents"
+    const val threeContinents = "Three Continents"
+    const val threeContinentsFair = "Three Continents Fair"
```

Comment:
This might create confusion. Why keep `Three Continents` if one of the continent has a clear advantage? 
I'd say ditch the `Three Continents` and rename the fair one to that, but lets see what other have to say. :)

Reply:
Agree, having an "unfair" mail doesn't sound like a good idea

Reply:
Done!